stdlib/format.mli

```diff
@@ -488,6 +488,18 @@ val safe_set_geometry : max_indent:int -> margin:int -> unit
    @since 4.08.0
 *)
 
+val pp_update_geometry : formatter -> (geometry -> geometry) -> unit
```

Comment:
This is the only function that doesn't have a a short variant `update_geometry`.

Reply:
Thanks! I fixed this by adding `update_geometry` as it should.